tree/ntuple/v7/inc/ROOT/RNTupleWriteOptions.hxx

```diff
@@ -41,6 +52,12 @@ public:
       kDefault,
    };
 
+   static constexpr std::uint64_t kDefaultMaxKeySize = 0x4000'0000; // 1 GiB
+
+   // clang-format off
```

Comment:
Does:
```suggestion
   // clang-format off
   static constexpr std::uint64_t kDefaultMaxKeySize = 0x4000'0000; // 1 GiB
   // clang-format on

```
work? (and would be clearer as it would be hiding the problematic code rather than some 'random' later code)

Reply:
I just tried and sadly no... it indents it this way:

```c++
   // clang-format off
   static constexpr std::uint64_t kDefaultMaxKeySize = 0x4000'0000; // 1 GiB
      // clang-format on

      friend Internal::RNTupleWriteOptionsManip;

```